roles/unicorn_user/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+--- # Setup an unprivileged user - needs to be run as root.
+- name: create unprivileged user
+  user: name="{{ unicorn_user }}" password="{{ unicorn_user_pass }}" append=yes state=present shell=/bin/bash
```

Comment:
Better to setup the unicorn user without password so that a login is not possible unless you modify .ssh/authorized_keys.